src/main/java/seedu/ddd/model/event/Event.java

```diff
@@ -0,0 +1,112 @@
+package seedu.ddd.model.event;
+
+import static seedu.ddd.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.ddd.commons.util.ToStringBuilder;
+import seedu.ddd.model.contact.client.Client;
+import seedu.ddd.model.contact.vendor.Vendor;
+
+/**
+ * Represents a Event in the address book.
+ * Guarantees: to be determined.
+ */
+public class Event {
+
+    private final ArrayList<Client> clients;
+    private final ArrayList<Vendor> vendors;
```

Comment:
Maybe we can use `Set` instead of `List`? We shouldn't need to insert duplicate contacts into `Event`.

Reply:
@leslieyip02 I think we can use `List`? The indices make it easier to access and modify contacts in `Event`.